pypeit/core/skysub.py

```diff
@@ -46,38 +48,47 @@ def skysub_npoly(thismask):
     return npoly
 
 
-def global_skysub(image, ivar, tilts, thismask, slit_left, slit_righ, inmask=None, bsp=0.6, sigrej=3.0, maxiter=35,
-                  trim_edg=(3,3), pos_mask=True, show_fit=False, no_poly=False, npoly=None):
+def global_skysub(image, ivar, tilts, thismask, slit_left, slit_righ, 
+                  inmask=None, bsp=0.6, sigrej=3.0, maxiter=35,
+                  trim_edg=(3,3), pos_mask=True, show_fit=False, 
+                  no_poly=False, npoly=None):
     """
     Perform global sky subtraction on an input slit
+    THIS NEEDS MORE DESCRIPTION
 
     Args:
-        image: float ndarray, shape (nspec, nspat)
-            Frame to be sky subtracted
-        ivar: float ndarray, shape (nspec, nspat)
-            Inverse variance image
-        tilts: float ndarray, shape (nspec, nspat)
-            Tilts indicating how wavelengths move across the slit
-        thismask : numpy boolean array, shape (nspec, nspat)
-            Specifies pixels in the slit in question
-        slit_left: ndarray of shape (nspec, 1) or (nspec)
+        image (`numpy.ndarray_`): 
```

Comment:
`numpy.ndarray_` --> `numpy.ndarray`_
for most of these Args

Reply:
done